### PR TITLE
[Multitask] Establishing baseline implementation for experimenting with model improvements

### PR DESCRIPTION
**Pull request by tnichols89**
- Overall model accuracy now modeled as vector norm
- Overall model training loss now modeled as vector norm
- Added support for writing per-epoch metrics out to a log file
- Added support for computing Pearson and Spearman correlation coefficients for STS
- Per-task loss values are now properly averaged using the length of the labels rather than `args.batch_size`
- We no longer evaluate model performance over the test data set on every epoch
- Removed WSL2-specific env since the Linux version works fine
- Updated Linux server and local conda env files to include `scikit-learn` and `torchmetrics`
- Added a variety of more helpful log output

### CODE REVIEW

**Comment by prushto** [comment]
Looks good!
